src/vivarium_census_prl_synth_pop/utilities.py

```diff
@@ -609,9 +610,7 @@ def record_metadata_proportions(final_output_dir: Path) -> None:
             melted_metadata["value"] / melted_metadata["number_of_rows"]
         )
         # Drop unnecessary columns
-        melted_metadata = melted_metadata.drop(
-            columns=["variable", "value", "number_of_rows"]
-        )
+        melted_metadata = melted_metadata.drop(columns=["variable", "value"])
```

Comment:
Why are you dropping the number of rows?

Reply:
Im NOT dropping the number of rows so we can weight the proportions for SSA later on in pseudopeople.